src/applications/vaos/components/FormButtons.jsx

```diff
@@ -19,9 +18,8 @@ export default function FormButtons({
       <div className="vaos__form-button-back mobile:vads-u-padding-right--1p5 medium-screen:vads-u-padding-right--0p5">
         <ProgressButton
           onButtonClick={onBack}
-          buttonText={backButtonText || 'Back'}
-          buttonClass="usa-button-secondary"
-          beforeText={typeof backBeforeText === 'string' ? backBeforeText : '«'}
```

Comment:
`backBeforeText` was never used and moving the chevron into `buttonText` fixes the mismatch between "Back" and "Continue" buttons